README.md

```diff
@@ -56,21 +56,10 @@ We are using webpack as a build tool.  See `webpack.config.js` for build depende
 `npm start` will spin up express directly.
 The express server is available on at [http://localhost:8000](http://localhost:8000).
 
->>>>>>> Update README and remove references to grunt
 ### Linter for JavaScript
 
 `npm run eslint`
 
-#### Generate a list of all eslint errors
```

Comment:
I should have removed this section of README when I removed eslint-takeoff package.